### ISSUE TITLE
Bad WebIO package version 0.8.93

### ISSUE THREAD

**Issue by chipkent** [JuliaGizmos/WebIO.jl#454]
Some Julia package sites have a bad WebIO version (`0.8.93`), while the current repo version is only `0.8.15`.  `Pkg` attempts to pull the `0.8.93` version and fails.

It can be seen here:  https://juliahub.com/ui/Packages/WebIO/v9IED/0.8.93?t=0

The problem can be avoided by forcing the version to `0.8.15` by doing `add WebIO@0.8.15`, but I'm sure it is causing problems for users.

**Comment by pkofod** [comment]
what package is this? It's really strange, I hit this error

**Comment by KristofferC** [comment]
>     Updating registry at `~/.julia/registries/JuliaComputingRegistry`

That registry probably has the "rogue" version.

**Comment by pkofod** [comment]
> That registry probably has the "rogue" version.

Yes, so the problem can occur if you have a JuliaPro-patched WebIO (so if you are still using Julia pro or specifically added that registry). Good to know, had me a bit confused :)
